php/Terminus/Commands/SiteCommand.php

```diff
@@ -5,6 +5,7 @@
 use Terminus;
 use Terminus\Auth;
 use Terminus\Request;
+use Terminus\Session;
 use Terminus\Utils;
 use Terminus\Commands\TerminusCommand;
 use Terminus\Helpers\Input;
```

Comment:
I can't put a comment on the line, but the `use Terminus\Models\User;` statement is obsolete here, too.